website/screens/components/dialog/specs/DialogSpecsPage.tsx

```diff
@@ -0,0 +1,476 @@
+              <th>Element</th>
+              <th>Core token</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>overlayOpacity</Code>
+              </td>
+              <td>Overlay</td>
+              <td>-</td>
+              <td>0.7</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>Property</th>
+              <th>Element</th>
+              <th>Core token</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>width</Code>
+              </td>
+              <td>Overlay</td>
+              <td>-</td>
+              <td>100vw</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>height</Code>
+              </td>
+              <td>Overlay</td>
+              <td>-</td>
+              <td>100vh</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <HeadingLink level={4}>Padding</HeadingLink>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>Padding</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>xxsmall</Code>
+              </td>
+              <td>6px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>xsmall</Code>
+              </td>
+              <td>16px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>small</Code> <em>(default)</em>
+              </td>
+              <td>24px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>medium</Code>
+              </td>
+              <td>36px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>large</Code>
+              </td>
+              <td>48px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>xlarge</Code>
+              </td>
+              <td>64px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>xxlarge</Code>
+              </td>
+              <td>100px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcText as="p">
+          And also apply different values to each side of the component: <Code>top</Code>,{" "}
+          <Code>bottom</Code>, <Code>left</Code> and <Code>right</Code>
+        </DxcText>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Accesibility</HeadingLink>
+        <HeadingLink level={4}>WCAG</HeadingLink>
+        <DxcList>
+          <DxcText>
+            Understanding WCAG 2.2 -{" "}
+            <DxcLink href="https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap.html" text="2.1.2 No Keyboard Trap" />
```

Comment:
Add to external links `newWindow` prop, please.

Reply:
it is done!